### DISCUSSION TITLE
Heimdall failing to get Radarr status

### DISCUSSION THREAD

**Discussion by ratnose** [linuxserver/Heimdall#741] [App issues]
Since Radarr updated to API v3 Heimdall gives me this error:
"Failed: Please make sure your URL is correct and that there is a trailing slash"
I have tried adding a slash same issue.

I am running everything inside a docker-compose setup, Heimdall v2.2.3 Radarr v4.0.0.5146

**Comment by damda58** [comment]
Hello,

Have you checked that the update did not generate a new token?

**Comment by cliffordwhansen** [comment]
@ratnose are you sure about those versions.

I am running Heimdall v2.2.2 and Radarr v3.2.2.5080 and it's working fine.

**Comment by EL-PEEPIO** [comment]
Same problem here, checked API token, URL and even on ip-adres but it keeps giving the error stated above.

Heimdall version V2.2.2 Radarr V4.0.0.5213

it seems to me it is in radarr but not 100% sure.

**Comment by micahmo** [comment]
Same issue here with Heimdall 2.2.2 and Radarr 4.0.0.5342.

**Comment by BzowK** [comment]
Same here...

**Comment by gravyflex** [comment]
Same issue with v2.2.3

**Comment by 1mintee1** [comment]
Any update on this?  Running Heimdall 2.4.11 with Radarr 4.1.0.6175.  

Still having this issue.

**Comment by firrae** [comment]
This still seems to be an issue, but is it with LSIO or Heimdall?

**Reply by firrae** [reply]
I added on to this issue (https://github.com/linuxserver/Heimdall/issues/1266) but it seems my issue stems from Heimdall not accepting IP address based URLs for Radarr and Sonarr at least.

**Comment by BayIsland757** [comment]
I had read that no status on the Radarr box (when the API token has been registered) may mean that the PHP memory is set too low.  Someone offered the suggestion on a Synology NAS using WebStation to adjust PHP MEMORY_LIMIT to above 512M.  I did this to no avail.   Does anyone have any insight on the issue of no status on Radarr?  (SAB, Lidarr and Sonarr all work properly)
